packages/terra-application/src/application-loading-overlay/ApplicationLoadingOverlayProvider.module.scss

```diff
@@ -0,0 +1,5 @@
+:local {
+  .container {
+    height: 100%;
```

Comment:
semi-related, but shouldn't the terra overlay container always be at height 100%? it seems weird we have to set this

Reply:
I can see it being required in most cases but not all. If the OverlayContainer decides to invert its height handling at some point, we can remove this.